### ISSUE TITLE
[React Native Gallery -> Sketch]: Property label name and control type is not defined to the 'Eraser', 'Undo', 'Clear' and Colors buttons.

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#307]
**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to ‘Sketch’ button and invoke it.
Now navigate to 'Eraser', 'Undo', 'Clear' and Colors using tab key and listen to the narrator announcement. 

**Actual Result:**
Property label name and control type is not defined to the 'Eraser', 'Undo', 'Clear' and Colors buttons. When user navigate to the 'Eraser', 'Undo', 'Clear' and Colors buttons, narrator is announcing as "Customized".

Note: Same issue is observed in TextToSppech, TrackPlayer cards also. 

**Expected Result:**
Proper label name and control type should be defined to 'Eraser', 'Undo', 'Clear' and Colors buttons. When user navigate to 'Eraser' button narrator should announce as "Eraser, button" and etc. 

**User impact:**
Users with visual impairment who rely on screen reader will not be able to know the purpose of the button if proper label name and control type is not defined. 

Internal bug link: https://dev.azure.com/microsoft/OS/_workitems/edit/43367266

**Comment by chrisglein** [comment]
These buttons:
![image](https://user-images.githubusercontent.com/26607885/223201408-ded0482d-aff5-4b7d-8afd-4f51724c66f3.png)
